tests/test_examples.py

```diff
@@ -157,6 +157,7 @@ def test_svm_batch(self):
     ('stump_regression_batch', None, None, (2020, 1)),
     ('svm_multiclass_batch', 'svm_multiclass_batch.csv', lambda r: r[0].prediction),
     ('univariate_outlier_batch', 'univariate_outlier_batch.csv', lambda r: r[1].weights),
+    ('dbscan_batch', None, None, (2020, 0)),
```

Comment:
This is not sufficient. The built-in test needs to check more, like the actual assignments.